tests/integration/component.go

```diff
@@ -46,7 +46,7 @@ func componentTests(args ...string) {
 			Expect(applicationName).To(Equal("app"))
 
 			// clean up
-			helper.CmdShouldPass("odo", "component", "delete", componentName, "-f")
+			// helper.CmdShouldPass("odo", "component", "delete", componentName, "-f")
```

Comment:
Out of curiosity, why has this been removed? Is it because by default `odo component delete` removes the app as well?

Reply:
Yeah, this shouldn't be disabled / removed

Reply:
what is app ? it is composed of `components`, if we delete the component, there is no purpose of application.
In this case, there is only one component in an application. Hence if we delete the component, there's no application.
Please try out the example locally and let me know

Reply:
and anyway we are deleting application in next line

Reply:
https://github.com/openshift/odo/pull/1728#pullrequestreview-239302561